### PR TITLE
feat(onboarding): jump to setup project

### PR DESCRIPTION
**Pull request by scefali**
This PR adds logic to jump from the welcome screen to instrumenting the application after creating a default project based on the query params. It's a follow up to this: https://github.com/getsentry/getsentry/pull/9547

### CODE REVIEW

**Review by JonasBa — APPROVED** [review]
Minor comment, but not blocking

**Comment by scefali** [comment]
@JonasBa What's your minor comment? Just "minor comment"? lol

**Comment by JonasBa** [comment]
Ahhh lol I didnt click add review comment 😭 , one sec

File: static/app/views/onboarding/index.tsx [hunk 1]
```diff
@@ -191,6 +191,14 @@ function Onboarding(props: Props) {
     );
   };
 
+  const jumpToSetupProject = useCallback(() => {
+    const nextStep = onboardingSteps.find(({id}) => id === 'setup-docs');
+    if (!nextStep) {
```
Comment (JonasBa) [inline static/app/views/onboarding/index.tsx:196]: Do we expect this to ever happen? If we dont, I would opt to throw here so we dont silently break the user flow (e.g. user clicking on a button that does nothing)

**Review by roggenkemper — APPROVED** [review]

File: static/app/views/onboarding/index.tsx (hunk 1 above)
Comment (scefali) [reply static/app/views/onboarding/index.tsx:196]: @JonasBa It shouldn't happen...but silently breaking here is preferable. It just means the user has to do project selection. It's better than blowing up to the user.

Comment (scefali) [reply static/app/views/onboarding/index.tsx:196]: It's not from a button click fwiw

Comment (JonasBa) [reply static/app/views/onboarding/index.tsx:196]: Ah yes, my bad.

**Pull request merged**